garnett/trajectory.py

```diff
@@ -660,14 +634,7 @@ def image(self):
 
     @image.setter
     def image(self, value):
-        try:
-            value = np.asarray(value, dtype=np.int32)
-        except ValueError:
-            raise ValueError("Images can only be set to numeric arrays.")
-        if not np.all(np.isfinite(value)):
-            raise ValueError("Images being set must all be finite numbers.")
-        elif not len(value.shape) == 2 or value.shape[1] != 3:
-            raise ValueError("Input array must be of shape (N,3) where N is the number of particles.")
+        value = self._validate_input_array(value, dim=2, nelem=3)
```

Comment:
This would potentially create a bug by attempting to convert the image data to floating point values. The data type of images is int32. I will fix that.

Reply:
Nice catch, this was my fault. Thanks for fixing